arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -715,6 +795,64 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Logic/workflows/{workflowName}/versions/{versionId}/triggers/{triggerName}/listCallbackUrl": {
+      "post": {
```

Comment:
@balajikris - This is fine. Calling it list* for a POST also allows it to be called from the ARM template.